homeassistant/auth/models.py

```diff
@@ -27,6 +28,9 @@
 TOKEN_TYPE_LONG_LIVED_ACCESS_TOKEN = "long_lived_access_token"
 
 
+AuthFlowResult = FlowResult[tuple[str, str]]
```

Comment:
```suggestion
AuthFlowResult = FlowResult[tuple[str, str]]
```
I would suggest to remove one empty line above (unless ruff requires it).